dotnet/src/Experimental/Process.Runtime.Dapr/Actors/ProcessActor.cs

```diff
@@ -17,14 +17,14 @@ namespace Microsoft.SemanticKernel;
 
 internal sealed class ProcessActor : StepActor, IProcess, IDisposable
 {
-    private const string DaprProcessInfoStateName = "DaprProcessInfo";
+    private const string DaprProcessInfoStateName = nameof(DaprProcessInfo);
+    private const string StepStateActivated = "kernelStepActivated";
     private const string EndStepId = "Microsoft.SemanticKernel.Process.EndStep";
```

Comment:
[not for this PR] Wondering if "global strings" like this one should be in a separate shared file, I imagine there will be more special internal components to validate when traversing steps in a process

Reply:
I absolutely like the discoverability aspect of centralizing these definitions.